core/src/main/java/io/temporal/samples/taskinteraction/client/ListOpenTasks.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (c) 2020 Temporal Technologies, Inc. All Rights Reserved
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.samples.taskinteraction.client;
+
+import static io.temporal.samples.taskinteraction.client.StartWorkflow.WORKFLOW_ID;
+
+import io.temporal.client.WorkflowClient;
+import io.temporal.samples.taskinteraction.Task;
+import io.temporal.samples.taskinteraction.TaskClient;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+import java.util.List;
+
+public class ListOpenTasks {
```

Comment:
I agree, but also I think it could be useful to show the workflow execution internal state. Do you think we can expose this information in any other way or should I just remove it?

Reply:
`ListOpenTasks` removed, 

I am maintaining the method `getOpenTask` to query the internal state and complete the task automatically from [UpdateTask ](https://github.com/temporalio/samples-java/pull/567/files#diff-c7b4940f7928e4030c177f127608dce85fbfd270934dc427ed0f4dc2785bb5e0R49)

I can remove the method `getOpenTask`, and have the dev running the example passing the task id to [UpdateTask](https://github.com/temporalio/samples-java/pull/567/files#diff-c7b4940f7928e4030c177f127608dce85fbfd270934dc427ed0f4dc2785bb5e0R49) , instead of getting the taskid from the workflow execution using `getOpenTask`